src/styles.css

```diff
@@ -531,13 +531,6 @@ span.tooltip {
   overflow: auto !important;
 }
 
-/* to manage icon color based on the parent color */
 .button .bi {
   color: inherit !important;
-}
-
-.dropdown-menu .dropdown-item:hover .icon i {
-  color: white !important;
```

Comment:
Will this cause issues at other places? Like selection of dropdown menu icon or hover?